logging/logging.go

```diff
@@ -9,6 +9,11 @@ import (
 var globalLogLevel = logrus.InfoLevel
 var globalLogLevelLock = sync.Mutex{}
 
+func GetProjectLogger() *logrus.Entry {
+	logger := GetLogger("")
+	return logger.WithField("name", "go-commons")
+}
```

Comment:
The logger should be provided by the caller, so that the `name` field can be set to the top level app. Otherwise, if you have multiple loggers using `go-commons` on a system, you won't be able to tell which CLI the log is coming from.